src/operations/get_more.ts

```diff
@@ -25,7 +29,10 @@ export class GetMoreOperation extends AbstractOperation {
 
   constructor(ns: MongoDBNamespace, cursorId: Long, server: Server, options: GetMoreOptions = {}) {
     super(options);
-    this.options = options;
+
+    // comment on getMore is only supported for server versions 4.4 and above
+    this.options = maxWireVersion(server) < 9 ? disallowOptions(options, ['comment']) : options;
```

Comment:
does the spec say that we have to silently ignore the option on servers below 4.4?

Reply:
the spec is kind of vague on this account, it just says "do not send comment", not "warn" or anything like that; so I think in this case it's fine to just delete